docs/ilm.asciidoc

```diff
@@ -59,6 +60,40 @@ apm-server:
 
 ILM can now be customized via the remaining `apm-server.ilm.setup.*` configuration options.
 
+[float]
+[[custom-ilm-alias]]
+==== Create a custom ILM rollover_alias
+
+You can define a custom `rollover_alias` per event type.
+
+NOTE: Ensure that the configured aliases for all event types start with the same prefix,
+and that this prefix is also configured for `setup.template.name` and `setup.template.pattern`.
+If the template configuration is not accordingly changed, the Elasticsearch index template with
+the predefined mappings will not match against created indices, leading to indexing issues.
+
+The example below shows how to change the rollover alias to some custom value,
+and setting the template configuration to a matching name and pattern.
+
+[source,yml]
+----
+apm-server:
+  ilm:
+    enabled: "auto"
+    setup:
+      mapping:
+        - event_type: "error"
+          rollover_alias: "apm-dev-error"
+        - event_type: "span"
+          rollover_alias: "apm-dev-span"
+        - event_type: "transaction"
+          rollover_alias: "apm-dev-transaction"
+        - event_type: "metric"
+          rollover_alias: "apm-dev-metric"
+
+setup.template.name: "apm-dev"
+setup.template.pattern: "apm-dev*"
```

Comment:
Actually variable parts are not allowed in `rollover_alias`. The indices and rollover aliases need to be set up on startup of the APM Server. Therefore allowed variables are restricted to known `observer.*` attributes. 
In case a variable part is configured, the server refuses to start up.

Reply:
Added a note to the docs.

Reply:
Ah right - thanks.